docs/source/admin-guide/certificate.md

```diff
@@ -143,9 +143,11 @@
                         sidecar.istio.io/inject: "false"                  
                     spec:
                       restartPolicy: Never
+                      imagePullSecrets:
+                      - name: regcred
                       containers:
                         - name: web-key-rotation
-                          image: gluufederation/certmanager:4.4.0-1
+                          image: gluufederation/certmanager:4.5.0-1
```

Comment:
Please use latest stable version from 4.5.2. Consult to Releases page for each `docker-*` repos.